scripts/package-torch.sh

```diff
@@ -7,17 +7,22 @@
 ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. && pwd)
 set -eux
 
-cd "$ROOT_DIR"
-
-VERSION=$(cat metatensor-torch/VERSION)
+VERSION=$(cat "$ROOT_DIR/metatensor-torch/VERSION")
 ARCHIVE_NAME="metatensor-torch-cxx-$VERSION"
 
-./scripts/n-commits-since-last-tag.py "metatensor-torch-v" > metatensor-torch/cmake/n_commits_since_last_tag
-tar cf "$ARCHIVE_NAME".tar metatensor-torch
-rm -f metatensor-torch/cmake/n_commits_since_last_tag
+TMP_DIR=$(mktemp -d)
+mkdir "$TMP_DIR/$ARCHIVE_NAME"
+
+cp -r "$ROOT_DIR"/metatensor-torch/* "$TMP_DIR/$ARCHIVE_NAME/"
+cp "$ROOT_DIR/LICENSE" "$TMP_DIR/$ARCHIVE_NAME"
+cp "$ROOT_DIR/AUTHORS" "$TMP_DIR/$ARCHIVE_NAME"
+./scripts/n-commits-since-last-tag.py "metatensor-torch-v" > "$TMP_DIR/$ARCHIVE_NAME/cmake/n_commits_since_last_tag"
```

Comment:
maybe can you add a small comment that the produced file is used during building?